### ISSUE TITLE
A consultation on the ACAS X algorithm in DO-385

### ISSUE THREAD

**Issue by 18120251** [dieman95/AcasXu#2]
Hello, have you ever reproduced the ACAS X algorithm implemented in the Julia language in DO-385? I encountered difficulties in the process of implementation.

**Comment by 1054651112** [comment]
Hello, have you solved the problem?
The Julia version given in the standard is too old. I used a new version of Julia to run the program, but there are still some testcases that are inconsistent with the standard. May I ask if you have these questions? Looking forward to your reply.

**Comment by 18120251** [comment]
> Hello, have you solved the problem? The Julia version given in the standard is too old. I used a new version of Julia to run the program, but there are still some testcases that are inconsistent with the standard. May I ask if you have these questions? Looking forward to your reply.

Yes. I replicated the ACAS-X algorithm using Julia version 1.4, but the TRM result report I ran always does not match the standard result.

**Comment by 1054651112** [comment]
Thank you for your reply. The version I am using is 1.9.2, and there is not much difference in TRMREPORT. My current issue is the inconsistency between STMREPORT and the standard, manifested in the 'belief_horiz' data in the 'intruder' attribute.

**Comment by 1054651112** [comment]
I haven't found the reason yet. I'm not sure if it's a problem with the data provided by the standard or with my input processing. Because in the testcases of Group 6, some of the output results were correct, but  there are differences in 'x_ rel' and 'dx_rel' at some moments compared to the standard. May I ask if you know the reason?
